src/RecyclableMemoryStream.cs

```diff
@@ -978,17 +995,17 @@ public override long Seek(long offset, SeekOrigin loc)
             int newPosition;
             switch (loc)
             {
-            case SeekOrigin.Begin:
-                newPosition = (int)offset;
-                break;
-            case SeekOrigin.Current:
-                newPosition = (int)offset + this.position;
-                break;
-            case SeekOrigin.End:
-                newPosition = (int)offset + this.length;
-                break;
-            default:
-                throw new ArgumentException("Invalid seek origin", nameof(loc));
+                case SeekOrigin.Begin:
```

Comment:
Looks like your editor is making some different spacing decisions than were in here previously. Can you revert all these lines that aren't part of the main change?

Reply:
Reverted!

You should add an [.editorconfig](https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options) file to the solution to prevent these unvoluntary changes.